src/gridfs-stream/download.ts

```diff
@@ -195,7 +195,9 @@ export class GridFSBucketReadStream extends Readable {
 
 function throwIfInitialized(stream: GridFSBucketReadStream): void {
   if (stream.s.init) {
-    throw new Error('You cannot change options after the stream has entered flowing mode!');
+    throw new MongoDriverError(
+      'You cannot change options after the stream has entered flowing mode!'
```

Comment:
clean up request, Can we rephrase this to something like `Options cannot be changed after stream is initialized`?